### ISSUE TITLE
Reconsider Job `emailChecker`

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/open-saas#143]
Is this job really needed? Seems a bit like redundant functionality. We do have `job dailyStatsJob` to showcase jobs.

I would consider dropping `job emailChecker` from open-saas.

Or, let’s at least make it a bit more interesting / practical: let’s make it send weekly summary to each user, of their activities. And give it a better name then also, like emailWeeklySummary. Yeah I can see myself using that. is that easy to make without knowing more about the app though?

Btw what if users don’t see that they have this job in their app → I can easily imagine them not noticing (it is at the end of the main.wasp file and you don't see its effects unless it hits Monday at 7am), and then in production email suddenly gets sent to users!
So maybe by default this should be turned off somehow? Or we should delete it hm. I like the functionality being there if we need it, but it is a bit much / can cause problems.

TLDR: I would either delete it, or make it more practical + make sure they have to somehow enable it and not have it working by default.

**Comment by vincanger** [comment]
@Martinsos I extracted this job in the recent vertical code organization leftovers to `src/newslertter` . I also renamed it to `checkAndQueueNewsletter`, which corresponds to a `user.sendNewsletter` property. At the moment, it's inactive and just exists in the codebase as an additional example of using recurring jobs with the email sender.
